rest_api.py

```diff
@@ -12,6 +12,25 @@
 is_filling = int(os.environ['FILL'])
 
 
+class InvalidData(Exception):
+    def __init__(self, message):
+        Exception.__init__(self)
+        self.message = message
+        self.status_code = 515

```

Comment:
Why 515 code is used? Maybe 4XX code is more suitable?

Reply:
done!